Formula/stanc3.rb

```diff
@@ -0,0 +1,38 @@
+class Stanc3 < Formula
+  desc "Stan transpiler"
+  homepage "https://github.com/stan-dev/stanc3"
+  # git is needed for dune subst
+  url "https://github.com/stan-dev/stanc3.git",
+      tag:      "v2.29.2",
+      revision: "2c254b3b11a4dbb82395f016886b557201dad130"
+  license "BSD-3-Clause"
+
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
+
+  uses_from_macos "unzip" => :build
+
+  def install
+    Dir.mktmpdir("opamroot") do |opamroot|
```

Comment:
Since builds are already in a tmpdir, is this still needed?

Reply:
Moved it to a directory in the buildpath (existing formulas with `opam` use either approach)

Reply:
Thanks, in that case this might also work:
```rb
mkdir "opamroot" do |opamroot|
```

Reply:
Hmm, not sure why it fails with the buildpath approach (but don't know much about opam or dune), so maybe tmpdir is needed. Tried the `mkdir` approach as well but ran into the same issue.